ansible_ai_connect/main/settings/legacy.py

```diff
@@ -60,8 +60,8 @@ def load_from_env_vars():
 
     # Model API Timeout (in seconds). Default is None.
     model_service_timeout = (
-        int(os.getenv("ANSIBLE_AI_MODEL_MESH_API_TIMEOUT"))
-        if os.getenv("ANSIBLE_AI_MODEL_MESH_API_TIMEOUT")
+        cast(int, os.getenv("ANSIBLE_AI_MODEL_MESH_API_TIMEOUT"))
```

Comment:
Are these still used?

They should be part of the JSON configuration.

Reply:
Yes, I had to, because `os.getenv()` return a string and pyright is not able (yet?) to take the `isnumeric()` into consideration to figure out that it's a `int`.